code/modules/projectiles/updated_projectiles/gun_system.dm

```diff
@@ -942,18 +942,14 @@ and you're good to go.
 		if(skill_accuracy)
 			gun_accuracy_mult += skill_accuracy * config.low_hit_accuracy_mult // Accuracy mult increase/decrease per level is equal to attaching/removing a red dot sight
 
-	projectile_to_fire.accuracy = round(projectile_to_fire.accuracy * gun_accuracy_mult) // Apply gun accuracy multiplier to projectile accuracy
-	projectile_to_fire.damage = round(projectile_to_fire.damage * damage_mult) 		// Apply gun damage multiplier to projectile damage
-	projectile_to_fire.damage_falloff	= round(projectile_to_fire.damage_falloff * damage_falloff_mult) 	// Apply gun damage bleed multiplier to projectile damage bleed
-	projectile_to_fire.projectile_speed += shell_speed_mod
-	projectile_to_fire.shot_from = src
-	projectile_to_fire.scatter += gun_scatter					//Add gun scatter value to projectile's scatter value
-
 	if(user) //The gun only messages when fired by a user.
 		gun_scatter += user.scatter_modifier //Any modifiers to scatter
 		projectile_to_fire.firer = user
 		if(iscarbon(user))
+			var/mob/living/carbon/C = user
 			projectile_to_fire.def_zone = user.zone_selected
+			if(C.stagger)
+				gun_scatter += 30
```

Comment:
Use a config here.

Reply:
Also put those calculations where they were before. The logic for the reader is modifiers-code first, then message/sounds code. It gets messier if you split the modifiers before and after.